react/SearchPage.js

```diff
@@ -1,19 +1,17 @@
-import React, {Component} from 'react'
-import PropTypes from 'prop-types'
+import React, { Component } from "react";
+import PropTypes from "prop-types";
 
 // eslint-disable-next-line
 export default class SearchPage extends Component {
   static propTypes = {
-    params: PropTypes.params,
-  }
+    params: PropTypes.object
+  };
 
   render() {
     return (
       <div className="pa4 ph7 pt0 georgia mw8 center f4 h-100">
-        <p>
-          {JSON.stringify(this.props)}
-        </p>
+        <p>{JSON.stringify(this.props)}</p>
```

Comment:
What is the use case for this?

Reply:
There isn't a use case really, this is the _**very**_ initial version of the new extension store.